packages/react/menu/src/Menu.tsx

```diff
@@ -244,9 +268,40 @@ const MenuRootContent = React.forwardRef((props, forwardedRef) => {
   }, []);
 
   return (
-    <MenuContentImpl {...props} ref={composedRefs} onDismiss={() => context.onOpenChange(false)} />
+    <MenuContentImpl
+      {...props}
+      ref={composedRefs}
+      // we make sure we're not trapping once it's been closed
+      // (closed !== unmounted when animating out)
+      trapFocus={context.open}
+      disableOutsidePointerEvents
```

Comment:
I think this is where https://github.com/radix-ui/primitives/pull/768/files?diff=split&w=1#r663963751 should move to now?

Reply:
I think so, but it looks like this change previously only applied to `ContextMenu`? I'm seeing some strangeness on `DropdownMenu` with this change so need to investigate.